src/services/resource-service.ts

```diff
@@ -167,10 +167,21 @@ export const ResourceService = {
   getResourcesCategoriesNames: (): Promise<
     AxiosResponse<CategoryNameInterface[]>
   > => axiosClient.get(URLs.resources.resourcesCategories.getNames),
+  getResourcesCategoriesName: () => {
+    return baseService.request<CategoryNameInterface[]>({
+      method: 'GET',
+      url: URLs.resources.resourcesCategories.getNames
+    })
+  },
   createResourceCategory: async (
-    params?: CreateCategoriesParams
-  ): Promise<AxiosResponse<Categories>> =>
-    await axiosClient.post(URLs.resources.resourcesCategories.post, params),
+    params: CreateCategoriesParams
+  ): Promise<Categories> => {
+    return await baseService.request<Categories>({
```

Comment:
Plese, remove async/await here, there is noting to await in this service
```suggestion
    return baseService.request<Categories>({
```